### PR TITLE
Use correct values for piping household names in LMS

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
Piped proxy name displaying incorrectly for the question 'What was [name's] main reason for coming to the UK in [year]?

[Card](https://trello.com/c/VBpix10B/2550-problem-displaying-piped-proxy-name)

### How to review 
Make sure name displays as expected.

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: data/cy/lms_2.json [hunk 1]
```diff
@@ -9853,7 +9853,7 @@
                         "questions": [{
                             "id": "no-primary-why-uk-question",
                             "titles": [{
-                                    "value": "Beth oedd prif reswm <em>{{ [answers['primary-household-member-first-name'][group_instance], answers['primary-household-member-last-name'][group_instance]] | format_household_name}}</em>  dros ddod i fyw yn y Deyrnas Unedig yn {{ answers['no-primary-first-arrive-in-uk-answer'][group_instance]|format_date }} ?",
+                                    "value": "Beth oedd prif reswm <em>{{ [group_instances[group_instance_id]['other-household-member-first-name-no-primary'], group_instances[group_instance_id]['other-household-member-last-name-no-primary'], group_instances[group_instance_id]['student-household-member-first-name-no-primary'], group_instances[group_instance_id]['student-household-member-last-name-no-primary']] | format_household_name }}</em>  dros ddod i fyw yn y Deyrnas Unedig yn {{ answers['no-primary-first-arrive-in-uk-answer'][group_instance]|format_date }} ?",
```
Comment (bitdivision) [inline data/cy/lms_2.json]: Do we need to add this change to the spreadsheet to keep everything up to date?

Comment (MebinAbraham) [reply data/cy/lms_2.json]: Done.

**Review by bitdivision — APPROVED** [review]

File: data/cy/lms_2.json (hunk 1 above)
Comment (jonnyshaw89) [inline data/cy/lms_2.json]: Are the double spaces intentional? `</em>  dros`

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: data/cy/lms_2.json (hunk 1 above)
Comment (MebinAbraham) [reply data/cy/lms_2.json]: Sorted

**Review by jonnyshaw89 — APPROVED** [review]

**Pull request merged**